<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapQuestConfigWidget</class>
 <widget class="QWidget" name="MapQuestConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>237</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="appKeyLabel">
     <property name="text">
      <string>AppKey</string>
     </property>
     <property name="buddy">
      <cstring>appKey</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="appKey"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Preference:</string>
     </property>
     <property name="buddy">
      <cstring>preference</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="preference"/>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="noMotorways">
     <property name="text">
      <string>Avoid motorways</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="noTollways">
     <property name="text">
      <string>Avoid tollways</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="noFerries">
     <property name="text">
      <string>Avoid ferries</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ascending slope:</string>
     </property>
     <property name="buddy">
      <cstring>ascending</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="ascending"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Descending slope:</string>
     </property>
     <property name="buddy">
      <cstring>descending</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="descending"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>An AppKey is required for MapQuest routing to work. You can register one &lt;a href=&quot;https://developer.mapquest.com/plan_purchase/steps/business_edition/business_edition_free/register&quot;&gt;here&lt;/a&gt;.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
